src/site/content/en/mini-apps/mini-app-alternative-runtime-environments/index.md

```diff
@@ -0,0 +1,264 @@
+---
+layout: post
+title: Other mini app runtime environments
+authors:
+  - thomassteiner
+date:  2021-02-05
+# updated: 2021-01-25
+description: |
+  This chapter presents a number of runtime environments for mini apps that are not mobile devices.
+tags:
+  - mini-apps
+---
+
```

Comment:
Intro here would be good too. I imagine you might expect most people to read the entire series, but if anyone wants to jump in and just read a page, it's helpful to provide a short intro that orients them

Reply:
I have solved this in https://github.com/GoogleChrome/web.dev/pull/3938/commits/9b1f6c5d6f25c7117132deef98f3c44472eca689 by adding an aside to each chapter apart the first that reads:

```
This post is part of an article collection where each article builds upon previous articles. If you just landed here, you may want to start reading from the beginning.
```